coral-schema/src/main/java/com/linkedin/coral/schema/avro/RelDataTypeToAvroType.java

```diff
@@ -110,9 +111,15 @@ private static Schema basicSqlTypeToAvroType(BasicSqlType relDataType) {
       case NULL:
         return Schema.create(Schema.Type.NULL);
       case TIMESTAMP:
-        Schema schema = Schema.create(Schema.Type.LONG);
+        schema = Schema.create(Schema.Type.LONG);
         schema.addProp("logicalType", "timestamp-millis");
         return schema;
+      case DATE:
+        // Spark recognizes the data type of {"type": "int", "logicalType": "date"} as a date type:
+        // https://github.com/apache/spark/blob/master/connector/avro/src/main/scala/org/apache/spark/sql/avro/AvroDeserializer.scala#L145
```

Comment:
Why is the Spark comment relevant? 
Especially, the code below converts a Calcite Date Type to Avro Date logical type.

Reply:
In our use case, Avro schema needs to be deserializer by Spark for upcast, but agreed that the Spark-related comment should not be added here since Coral-Schema is not just used for Spark.